app/middlewares/validation/index.js

```diff
@@ -12,7 +12,7 @@ class ValidationMiddleware {
   static validate(schema) {
     return async (req, res, next) => {
       try {
-        await validateInput(schema, req.body);
+        validateInput(schema, req.body);
```

Comment:
Even tho vscode says the use of await is redundant here, you might actually need it.